tools/generate-intrinsics-specs.sh

```diff
@@ -53,4 +53,6 @@ generate_rst_specs ./tools/intrinsic_db/mve.csv \
 
 # Check changes
 check_changes ./tmp/advsimd.new.rst ./neon_intrinsics/advsimd.rst
-#check_changes ./tmp/mve.new.rst ./mve_intrinsics/mve.rst
+check_changes ./tmp/mve.new.md ./mve_intrinsics/mve.md
+markdown-link-check --verbose --config package-lock.json mve_intrinsics/mve.md
+markdown-link-check --verbose --config package-lock.json morello/morello.md
```

Comment:
This doesn't make sense here - the Morello file is not generated by the specs. I am started to ponder if, instead of enabling the `markdown-link-check` on an individual file basis, we should instead activate it on the whole projects as github action, as explained here: https://github.com/marketplace/actions/markdown-link-check